python/ray/experimental/channel/nccl_group.py

```diff
@@ -66,11 +67,17 @@ def __init__(
             actor_handles: A list of actor handles, in rank order.
             cuda_stream: A raw CUDA stream to dispatch NCCL ops to. If rank is
                 specified, then this must be specified too.
+            dedicated_streams: Whether to use one dedicated stream for `send`
+                and another for `recv`. This is useful for overlapping communication
+                with computation.
+                If this is True, then `cuda_stream` is not used and new streams
```

Comment:
cuda_stream should still be used for compute right?

Reply:
Actually no. If this flag is true, a new execution stream will be created as well, rather than using `cuda_stream`. So aDAG uses all new streams different from outside.

Reply:
Hmm why is that? I feel natural this controls the compute stream. But anyway we will revamp this api to allow custom stream, so it may be okay..

Reply:
Updated doc.